package.json

```diff
@@ -5,31 +5,36 @@
   "main": "index.js",
   "scripts": {
     "tsc": "tsc",
-    "dev": "ts-node-dev index.ts",
+    "dev": "NODE_ENV=development ts-node-dev index.ts",
     "lint": "eslint --ext .ts .",
     "fix-lint": "eslint --fix --ext .ts .",
-    "start": "npm run tsc && node build/index.js",
+    "start": "NODE_ENV=development npm run tsc && node build/index.js",
```

Comment:
Ultimately here environment will be production.

Reply:
Correct, I left it like this temporary.